### PR TITLE
fix text and rect overlay bug

### PR DESCRIPTION
**Pull request by hendrik-weiler**

### CODE REVIEW

**Comment by tommycli** [comment]
Overall looks good. Let's make the name of the keys something more abstract like maybe: Menu, Special, Start, Select?

**Comment by tommycli** [comment]
And Maybe the keys bounds to stuff like Shift, Ctrl, Alt, Enter?

**Comment by hendrik-weiler** [comment]
Yes to more keys But The keys can be Different used for every game so it should not be fixed

**Comment by tommycli** [comment]
Remapping can be a separate feature

**Comment by tommycli** [comment]
But the keys should not be named after the actual keyboard mapping, as there are gamepads

**Comment by hendrik-weiler** [comment]
The keys should be mapped to gamepads somehow.

But i dont know anything about gamepads.

File: common/src/main/scala/rpgboss/player/InputMultiplexer.scala [hunk 1]
```diff
@@ -54,8 +54,12 @@ object MyKeys {
   val Right = 3
   val OK = 4
   val Cancel = 5
+  val A = 6
```
Comment (tommycli) [inline common/src/main/scala/rpgboss/player/InputMultiplexer.scala]: Call these something else. Maybe just "Special1", "Special2", etc.

File: common/src/main/scala/rpgboss/player/InputMultiplexer.scala [hunk 2]
```diff
@@ -65,6 +69,10 @@ object MyKeysEnum extends Enumeration {
   val Right = Value(3, "Right")
   val OK = Value(4, "OK")
   val Cancel = Value(5, "Cancel")
+  val A = Value(6, "A")
```
Comment (tommycli) [inline common/src/main/scala/rpgboss/player/InputMultiplexer.scala]: The default keyboard mappings should be something more accessible. Maybe Ctrl, Shift, Alt, Enter?

File: common/src/main/scala/rpgboss/player/InputMultiplexer.scala [hunk 3]
```diff
@@ -131,6 +139,10 @@ class InputMultiplexer extends InputAdapter {
     case Keys.M => Some(MyKeys.Cancel)
     case Keys.X => Some(MyKeys.Cancel)
     case Keys.ESCAPE => Some(MyKeys.Cancel)
+    case Keys.A => Some(MyKeys.A)
```
Comment (tommycli) [inline common/src/main/scala/rpgboss/player/InputMultiplexer.scala]: Change the default mappings to something more accessible. Maybe Ctrl, Shift, Alt, Enter?

Comment (hendrik-weiler) [inline common/src/main/scala/rpgboss/player/InputMultiplexer.scala]: Where do i find the list of keys? I just tried with A,W,S,D.

**Comment by hendrik-weiler** [comment]
With anti aliasing and fontsize regulation you can do nice ui elements:

unbenannt-1

File: common/src/main/scala/rpgboss/player/InputMultiplexer.scala (hunk 3 above)
Comment (tommycli) [inline common/src/main/scala/rpgboss/player/InputMultiplexer.scala]: http://libgdx.badlogicgames.com/nightlies/docs/api/com/badlogic/gdx/Input.Keys.html

**Pull request merged**